apps/minimal-app/configuration/nrf54l15dk_nrf54l15_cpuapp/images/mcuboot/ota-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Presumably we want to be able to OTA from the minimal app to something else - should we put the key in a shared folder or is your view that the first OTA needs to be signed by this key, and subsequent one using a separate key?